modules/standard/ChplConfig.chpl

```diff
@@ -25,10 +25,10 @@
 /*
 Access to Configuration information for the ``chpl`` compiler
 
-The values of Chapel's set and inferred configuration settings at
-compile-time are accessible through the parameters in this module.
-This information can also be displayed from the command line by
-executing the compiled program with the ``--about`` flag.
+This module's contents provide access to the compile-time values of
```

Comment:
> "This module's contents provide access to the compile-time values of Chapel's set and inferred configuration options."

The questions that comes to mind when I read this are:
  - what are configuration options?
  - what is a "set configuration option", what is an "inferred configuration option", are these terms I should know?
  - how would I set a configuration option?

For "what are configuration options?" question I think it'd be evident from the rest of the content on the doc page.

For the second question, I've used Chapel enough that I can guess that by "set options" you mean things the user set by environment variable or passed to the compiler using command line arguments.  But in the absence of my experience I could see finding that a little confusing.

I don't know if this makes it any clearer but to take a stab at it I think I'd say:

"This module's contents provide access to the compile-time values of Chapel's configuration options (whether they were set explicitly or assigned to default\inferred values)."

Followed up with something like

"For information about how to set these options see: [link to whatever page we have that talks about setting these as environment variables or passing them to the compiler]."

And then have the line about using the `--about` flag.

Reply:
@stonea: How do you feel about this rewrite:
<img width="764" alt="Screen Shot 2022-02-09 at 11 25 39 AM" src="https://user-images.githubusercontent.com/7536222/153275381-a35b1ac8-d21a-4206-adc4-d7e34aa23afa.png">

Reply:
I think that's good: a lot clearer without going in too much detail. Thanks!